tests/test_port_dpb_system.py

```diff
@@ -248,25 +242,380 @@ def test_port_breakout_with_acl(self, dvs):
+        intf_entries = tbl.getKeys()
+        # one loopback router interface
+        assert len(intf_entries) == 1
+
+        # Breakout Ethernet8 WITH "-l" option and ensure
+        # ip address gets configured as per default_config_db.json
+        dpb.verify_port_breakout_mode(dvs, "Ethernet8", breakoutMode1x)
+        dvs.change_port_breakout_mode("Ethernet8", breakoutMode2x, breakoutOption)
+        dpb.verify_port_breakout_mode(dvs, "Ethernet8", breakoutMode2x)
+
+        intf_entries = tbl.getKeys()
+        # one loopback router interface and one port based router interface
+        assert len(intf_entries) == 2
+
+        # check ASIC route database
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY")
+        for key in tbl.getKeys():
+            route = json.loads(key)
+            if route["dest"] == "10.0.0.8/31":
+                subnet_found = True
+            if route["dest"] == "10.0.0.8/32":
+                ip2me_found = True
+
+        assert subnet_found and ip2me_found
+
+        # Breakout Ethernet8 WITH "-f" option and ensure cleanup happened
+        dpb.verify_port_breakout_mode(dvs, "Ethernet8", breakoutMode2x)
+        dvs.change_port_breakout_mode("Ethernet8", breakoutMode1x, "-f")
+        dpb.verify_port_breakout_mode(dvs, "Ethernet8", breakoutMode1x)
+
+        # check ASIC router interface database
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ROUTER_INTERFACE")
+        intf_entries = tbl.getKeys()
+        # one loopback router interface
+        assert len(intf_entries) == 1
+
+        # check ASIC database
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY")
+        for key in tbl.getKeys():
+            route = json.loads(key)
+            if route["dest"] == "10.0.0.8/31":
+                assert False
+            if route["dest"] == "10.0.0.8/32":
+                assert False
+
+    def test_cli_command_negative(self, dvs):
+        dvs.setup_db()
+        dpb = DPB()
+        self.setup_db(dvs);
+
+        portGroup = ["Ethernet0", "Ethernet1", "Ethernet2", "Ethernet3"]
+        rootPortName = portGroup[0]
+        vlanIDs = ["100", "101"]
+        aclTableNames = ["DPB_ACL_TBL_1", "DPB_ACL_TBL_2"]
+        breakoutMode1x = "1x100G[40G]"
+        breakoutMode4x = "4x25G[10G]"
+
+        # Create only one ACL table and one VLAN table
+        bind_ports = []
+        self.dvs_acl.create_acl_table(aclTableNames[0], "L3", bind_ports)
+        self.dvs_vlan.create_vlan(vlanIDs[0])
+
+        # Add root port to ACL and VLAN tables
+        bind_ports = []
+        bind_ports.append(rootPortName)
+        self.dvs_acl.update_acl_table(aclTableNames[0], bind_ports)
+        self.dvs_vlan.create_vlan_member(vlanIDs[0], rootPortName)
+
+        # Breakout port WITHOUT "-f" option when dependencies exist
+        # TBD: Verify the list of dependencies returned by CLI command
```

Comment:
@praveen-li Could you please comment on how we can achieve this?

Reply:
@samaity : You use click.echo to print deps right?, In that case we need to parse screen output, not sure if any VS test has support for it.

Reply:
Yes. We can parse screen output or can keep the output in a temp file and then compare the file output to the expected output if that makes sense.

Reply:
Sounds like a good idea, so something like below:

```
deps = $expected_result
CLI > out_file
actual_result = $(cat out_file | grep $deps)
assert $actual_result == $expected_result
```

Reply:
OK, Let me take it up as followup task